src/pages/login_page.py

```diff
@@ -26,14 +26,20 @@ def open_login_page(self) -> None:
         logger.info(f"Opening page")
         self.page.goto(self.base_url)
 
-    # TODO: add check and additional CLEAR action
     def input_username(self, text) -> None:
```

Comment:
1. Is this approach OK?
2. Receive WARN in logs, but input should be empty. Looks like it has "spaces", but not sure.  Applies to `input_password()` too